packages/editor-ui/src/views/SettingsUsersView.test.ts

```diff
@@ -1,191 +1,283 @@
 import { within } from '@testing-library/vue';
 import userEvent from '@testing-library/user-event';
-import { createPinia, setActivePinia } from 'pinia';
 import { createComponentRenderer } from '@/__tests__/render';
-import { cleanupAppModals, createAppModals, getDropdownItems } from '@/__tests__/utils';
-import ModalRoot from '@/components/ModalRoot.vue';
-import DeleteUserModal from '@/components/DeleteUserModal.vue';
+import { getDropdownItems, mockedStore } from '@/__tests__/utils';
 import SettingsUsersView from '@/views/SettingsUsersView.vue';
-import { useProjectsStore } from '@/stores/projects.store';
 import { useUsersStore } from '@/stores/users.store';
 import { createUser } from '@/__tests__/data/users';
-import { createProjectListItem } from '@/__tests__/data/projects';
 import { useRBACStore } from '@/stores/rbac.store';
-import { DELETE_USER_MODAL_KEY, EnterpriseEditionFeature } from '@/constants';
-import * as usersApi from '@/api/users';
 import { useSettingsStore } from '@/stores/settings.store';
-import { defaultSettings } from '@/__tests__/defaults';
-import { ProjectTypes } from '@/types/projects.types';
+import { createTestingPinia, type TestingOptions } from '@pinia/testing';
+import { merge } from 'lodash-es';
+import { useUIStore } from '@/stores/ui.store';
+import { useSSOStore } from '@/stores/sso.store';
+
+const loggedInUser = createUser();
+const invitedUser = createUser({
+	firstName: undefined,
+	inviteAcceptUrl: 'dummy',
+	role: 'global:admin',
+});
+const user = createUser();
+const userWithDisabledSSO = createUser({
+	settings: { allowSSOManualLogin: true },
+});
+
+const initialState = {
+	users: {
```

Comment:
We have constants defined for store names which we should use. See `src/constants`.

```suggestion
	[STORES.USERS]: {
```